instaLooter/cli.py

```diff
@@ -161,15 +163,19 @@ def main(argv=None):
         return 0
 
     elif args['login']:
-        try:
-            args['--username'] = six.moves.input('Username: ')
-            login(InstaLooter(), args)
-            return 0
-        except ValueError as ve:
-            console.error(ve)
-            if args["--traceback"]:
-               traceback.print_exc()
-            return 1
+        for i in range(int(args['--try-count'])):
+            try:
+                args['--username'] = six.moves.input('Username: ')
+                login(InstaLooter(), args)
+                return 0
+            except ValueError as ve:
+                if i != int(args['--try-count']) - 1:
+                    console.warn('Wrong Username or Password')
+                    continue
+                console.error(str(ve) + ', the number of tries exceeded')
```

Comment:
Just show the message here, the ValueError text is not needed since it can be displayed with `--traceback` if needed.